js/app.ts

```diff
@@ -349,8 +348,10 @@ export class App {
             AnimationSettings.platform = "tablet";
         }
 
+        orientationService.enableRotation();
```

Comment:
This is very initial rotation, we should set landscape for ipad and then we suppress rotation. This is needed because of initialisation of pages before device ready event, someone is suppresing rotation before setting our desired orientation.